src/aws-code-deploy/orb.yml

```diff
@@ -211,10 +206,7 @@ jobs:
     steps:
       - checkout
       - aws-cli/install
-      - aws-cli/configure:
-          aws-access-key-id: $AWS_ACCESS_KEY_ID
-          aws-secret-access-key: $AWS_SECRET_ACCESS_KEY
-          region: << parameters.region >>
+      - aws-cli/configure
```

Comment:
I think the `aws-code-deploy` description should document that it expects `$AWS_ACCESS_KEY_ID`, `$AWS_SECRET_ACCESS_KEY`, and `$AWS_REGION` to be set. I'm not sure how discoverable these dependencies would be otherwise.